<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddConstant</class>
 <widget class="QDialog" name="DialogAddConstant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="const1Label">
     <property name="text">
      <string>Const1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="const1LineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="const2Label">
     <property name="text">
      <string>Const2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="const2LineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
